src/utilities/__tests__/astFromValue-test.ts

```diff
@@ -143,6 +168,11 @@ describe('astFromValue', () => {
       kind: 'NullValue',
     });
 
+    expect(astFromValue(9007199254740993n, GraphQLString)).to.deep.equal({
+      kind: 'StringValue',
+      value: '9007199254740993',
```

Comment:
```suggestion
      value: '9007199254740992',
```

There is a loss of precision when converting a bigint to a float